book/src/kernel/linux-compatibility/limitations-on-system-calls/file-and-directory-operations.md

```diff
@@ -9,7 +9,27 @@ utime, and utimensat
 under this category.
 -->
 
-## `open` and `openat`
+### `newfstatat`
+
+Supported functionality in SCML:
+
+```c
+// Retrieve file status by file descriptor
+newfstatat(
+    dirfd, path, statbuf,
+    flags = AT_EMPTY_PATH | AT_SYMLINK_NOFOLLOW
+);
+```
+
+Silently-ignored flags:
+* `AT_NO_AUTOMOUNT`
+
+For more information,
+see [the man page](https://man7.org/linux/man-pages/man2/newfstatat.2.html).
+
+## Basic File Operations
```

Comment:
`newfstatat` is just a variant of `stat`, which is also quite "basic" in my opinion. So the current way of grouping the system calls does not make sense. I suggest reverting all changes made to this file except changing `##` to `###`.